src/ride/serializers.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.models import User
+from path.serializers import PathSerializer
+from person.serializers import UserSerializer
+from rest_framework import serializers
+from ride.models import Ride
+
+
+class RideSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(source="path.id")
+    creator = serializers.SerializerMethodField("get_creator")
```

Comment:
I'd use a Person serializer instead of doing get methods like this for all of the serializers.

for example, for `get_riders`, I would use a user serializer and do `many=True`

here's an example: https://github.com/flickchicks/flick-backend/blob/master/src/group/serializers.py